sdk/core/azure_core/README.md

```diff
@@ -1,20 +1,283 @@
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let options = SecretClientOptions {
+        api_version: "7.0".to_string(),
+        client_options: ClientOptions::default(),
+    };
+
+    let credential = DefaultAzureCredential::new()?;
+    let client = SecretClient::new(
+        "https://your-key-vault-name.vault.azure.net/",
+        credential.clone(),
+        Some(options),
+    )?;
+
+    Ok(())
+}
+```
+
+### Accessing HTTP Response Details Using `Response<T>`
+
+_Service clients_ have methods that can be used to call Azure services. We refer to these client methods as _service methods_.
+_Service methods_ return a shared `azure_core` type `Response<T>` (in rare cases its non-generic sibling, a raw `Response`).
+This type provides access to both the deserialized result of the service call, and to the details of the HTTP response returned from the server.
+
+```rust no_run
+use azure_core::Response;
+use azure_identity::DefaultAzureCredential;
+use azure_security_keyvault_secrets::{models::SecretBundle, SecretClient};
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // create a client
+    let credential = DefaultAzureCredential::new()?;
+    let client = SecretClient::new(
+        "https://your-key-vault-name.vault.azure.net/",
+        credential.clone(),
+        None,
+    )?;
+
+    // call a service method, which returns Response<T>
+    let response: Response<SecretBundle> = client.get_secret("SecretName", "", None).await?;
+
+    // Response<T> has two main accessors.
+    // The into_body() function for accessing the deserialized result of the call
+    let secret = response.into_body().await?;
+
+        // get response again because it was moved in above statement
+    let response: Response<SecretBundle> = client.get_secret("SecretName", "", None).await?;
+
+    // .. and the deconstruct() method for accessing all the details of the HTTP response
+    let (status, headers, body) = response.deconstruct();
+
+    // for example, you can access HTTP status
+    println!("Status: {}", status);
+
+    // or the headers
+    for (header_name, header_value) in headers.iter() {
+        println!("{}: {}", header_name.as_str(), header_value.as_str());
+    }
+
+    Ok(())
+}
+```
+
+More on response types in [response samples](https://github.com/Azure/azure-sdk-for-rust/blob/main/sdk/core/azure_core/samples/Response.md).
+
+### Setting up console logging
+
+To create an Azure SDK log listener that outputs messages to console use `AzureEventSourceListener::create_console_logger` method.
+
+```rust no_run
+// Setup a listener to monitor logged events.
+let listener = AzureEventSourceListener::create_console_logger();
+```
+
+More on logging in [diagnostics samples](https://github.com/Azure/azure-sdk-for-rust/blob/main/sdk/core/azure_core/samples/Diagnostics.md).
+
+### Reporting Errors `RequestFailedException`
```

Comment:
As noted above, no exceptions.

Reply:
Altered section to be on Handling Result of Error